docs/reference/commandline/run.md

```diff
@@ -720,12 +720,12 @@ On Windows, `--isolation` can take one of these values:
 | Value     | Description                                                                                |
 |:----------|:-------------------------------------------------------------------------------------------|
 | `default` | Use the value specified by the Docker daemon's `--exec-opt` or system default (see below). |
-| `process` | Shared-kernel namespace isolation (not supported on Windows client operating systems).     |
+| `process` | Shared-kernel namespace isolation (not supported on Windows client operating systems older than Windows 10 1809).     |
```

Comment:
Small nit: we should update the table spacing in the description column

Reply:
Yes, we probably should've (to make the source a bit better readable 😅